jersey/src/main/java/org/pac4j/jax/rs/jersey/features/Pac4JValueFactoryProvider.java

```diff
@@ -165,69 +159,94 @@ public Binder() {
          *            a builder for a {@link ProfileManager}, can be <code>null</code> and default will be used.
          */
         public Binder(ProfileFactoryBuilder profile, OptionalProfileFactoryBuilder optProfile,
-                ProfileManagerFactoryBuilder manager) {
+                      ProfileManagerFactoryBuilder manager) {
             this.profile = profile == null ? ProfileValueFactory::new : profile;
             this.optProfile = optProfile == null ? OptionalProfileValueFactory::new : optProfile;
-            this.manager = manager == null ? ProfileManagerValueFactory::new : manager;
+            this.manager = manager;
```

Comment:
why don't you do the same as with profile and optProfile and initialize manager here?

Reply:
Because if the manager is not specified I wanted it to be injected. This would avoid the need for a Pac4J user to inject everything in the Feature and then having to pass it all to the Binder constructor.
It doesn't feel ideal to me because now the manager is special compared to the profile and optProfile, but I thought of it as a good compromise.

Reply:
The user shouldn't be using the Binder with parameters but only the binder without parameters.
And anyway, in any case, the constructor didn't change so I'm not sure what you mean ^^

Let's keep it like this, if it works it's ok, we will have other occasions to clean that, I'm pretty sure there is a lot that can be done for this class...